packages/venia-ui/lib/components/ProductOptions/__tests__/swatch.spec.js

```diff
@@ -20,8 +20,6 @@ const defaultProps = {
 test('renders a Swatch correctly', () => {
     const component = testRenderer.create(<Swatch {...defaultProps} />);
 
-    expect(() => component.root.findByType(Icon)).toThrow();
```

Comment:
Removed this since checkmarks will always be rendered in the DOM now, but with an opacity of 0.